coordinator/src/lib.rs

```diff
@@ -57,6 +57,14 @@ impl BlockExecutor for Coordinator {
         unimplemented!()
     }
 
+    fn prepare_transactions_with_result<'a>(
```

Comment:
It seems that it doesn't return the result.

Reply:
That's right, but as you can see, I actually haven't implemented any of the methods in the file yet.
The implementation would be heavily related to some coordinator-side decisions, so I have to discuss with @dynaxis and other team members.

I think we should implement them soon so I'll have a discussion and work on it, but not in this PR.

Reply:
Even you haven't implemented it yet, you can declare the result type. If you cannot decide how to implement the result type yet, you should not name this method as `prepare_transactions_with_result`.

Reply:
Oh, I misunderstood your concern.
You're worrying because using the term `result` without returning an explicit `Result` type is confusing, right?
Actually I didn't mean this method to return a result type.
The method does two things.
1. Choose transactions to be included in the next block and return them
2. Execute the chosen transactions on the given `context`.

Previously Dohyung suggested the name for a reason in Discord.
Could you suggest a better name based on its behavior?
Maybe `prepare_block` works, because it's easy to infer that this method execute transactions (One of the argument is a mutable reference of `StorageAccess`)

https://discordapp.com/channels/682525519711043609/683931185982079019/699187814977372225
> @byeongjee renaming remove_transactions to filter_transactions seems a good idea. And your idea of splitting methods to open_block, close_block and the others also sounds great. But naming-wise, select_and_execute prefix seems very straight on what it does, but looks a bit verbose to me. And we don’t even select and then execute Txes. Maybe prepare_block would be better, IMHO. But I’m not sure. The exact meaning is ‘select Txes for inclusion in a block to propose by ordering and executing Txes, and then return the root hash’. So if we are to name it to the meaning, it would be more like prepare_transactions_with_result.

Reply:
I used `prepare_block` for now.